core/src/main/java/apoc/nodes/NodesConfig.java

```diff
@@ -0,0 +1,30 @@
+package apoc.nodes;
+
+import apoc.util.Util;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class NodesConfig {
+
+    public static final String MAX_DEPTH_KEY = "maxDepth";
+    public static final String REL_TYPE_KEY = "relType";
```

Comment:
Should we allow a list of relationship types here? Is that allowed to make this a list?

Reply:
Of course. Changed with `relTypes`